onnxruntime/core/graph/graph_flatbuffers_utils.cc

```diff
@@ -167,12 +200,44 @@ Status LoadInitializerOrtFormat(const fbs::Tensor& fbs_tensor,
     ORT_RETURN_IF(nullptr == fbs_raw_data, "Missing raw data for initializer. Invalid ORT format model.");
 
     // fbs_raw_data is uint8_t vector, so the size is byte size
-    initializer.set_raw_data(fbs_raw_data->Data(), fbs_raw_data->size());
+    // however, we want to load int64_t types to int64_t data
+    if (initializer.data_type() == TensorProto_DataType_INT64) {
+      const auto num_elements = fbs_raw_data->size() / sizeof(int64_t);
```

Comment:
It may be more correct to fix CopySparseData to read indices from either int64_data or raw_data. That way we don't need any special casing here. I don't see anything in the ONNX spec that says indices must be in int64_data, so we should support them coming from raw data as well.